### PR TITLE
Added ultrahud to hud-db

### PR DESCRIPTION
**Pull request by cremator-2fort**
Hopefully this works.

### CODE REVIEW

**Review by mastercoms — CHANGES_REQUESTED** [review]
Looking good but you'll have to upload ultrahud-logo to hud-resources/ultrahud folder

**Comment by cremator-2fort** [comment]
Added the png to ultrahud-tf2

File: hud-data/ultrahud-tf2.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "resources": [
+    "ultrahud-logo"
+  ],
+  "flags": [
+    "menus",
+    "crosshairs"
+  ],
+  "repo": "https://github.com/cremator-2fort/ULTRAHUD-TF2",
+  "hash": "f9ec4efc0e673b26627e0a8cb2d301036c98a1c0",
+  "prerelease": false,
```
Comment (mastercoms) [inline hud-data/ultrahud-tf2.json:17]:
```suggestion
  "prerelease": true,
```

If it's early access, would you like to set prerelease to true?

Comment (cremator-2fort) [reply hud-data/ultrahud-tf2.json:17]: Yeah, thats fine.

**Comment by rbjaxter** [comment]
I don't believe it matters for submitting to comfig huds (correct me if I'm wrong @mastercoms) but you may want to move the contents of your hud up one directory so `ULTRAHUD-TF2` contains all your files/folders (`materials`, `resources`, `scripts`, etc), instead of having them within `ULTRAHUD-TF2/ultrahud/`

chrome_w4Pxz2fOqH

**Comment by mastercoms** [comment]
Yup it will make it list an incompatible

**Comment by cremator-2fort** [comment]
So I added an update which removes the `ultrahud` subfolder, so everything is now just in `ULTRAHUD-TF2`. Does this make it compatible?

**Review by mastercoms — APPROVED** [review]

**Pull request merged**